profanities.go

```diff
@@ -3,68 +3,397 @@ package goaway
 var profanities = []string{
 	"anal",
 	"anus",
+	"arrse",
 	"arse",
-	"ass",
-	"ballsack",
+	"assfucker",
+	"asses",
+	"assfucker",
+	"assfukka",
+	"asshole",
+	"assholes",
+	"asswhole",
+	"ballbag",
 	"balls",
+	"ballsack",
 	"bastard",
-	"bitch",
+	"beastial",
+	"beastiality",
+	"bellend",
+	"bestial",
+	"bestiality",
 	"biatch",
+	"bitch",
+	"bitcher",
+	"bitchers",
+	"bitches",
+	"bitchin",
+	"bitching",
+	"bloody",
 	"blowjob",
+	"blowjobs",
+	"boiolas",
 	"bollock",
 	"bollok",
 	"boner",
 	"boob",
+	"boobs",
+	"booobs",
+	"boooobs",
+	"booooobs",
+	"booooooobs",
+	"breasts",
+	"buceta",
 	"bugger",
 	"bum",
+	"bunnyfucker",
 	"butt",
+	"butthole",
+	"buttmunch",
+	"buttplug",
+	"cock",
+	"cocksucker",
+	"carpetmuncher",
+	"cawk",
+	"chink",
+	"cipa",
+	"clit",
 	"clitoris",
+	"clits",
+	"cnut",
 	"cock",
+	"cockface",
+	"cockhead",
+	"cockmunch",
+	"cockmuncher",
+	"cocks",
+	"cocksuck",
+	"cocksucked",
+	"cocksucker",
+	"cocksucking",
+	"cocksucks",
+	"cocksuka",
+	"cocksukka",
+	"cokmuncher",
+	"coksucka",
 	"coon",
 	"crap",
-	"cum",
+	"cummer",
+	"cumming",
+	"cums",
+	"cumshot",
+	"cunilingus",
+	"cunillingus",
+	"cunnilingus",
 	"cunt",
+	"cuntlick",
+	"cuntlicker",
+	"cuntlicking",
+	"cunts",
+	"cyalis",
+	"cyberfuc",
+	"cyberfuck",
+	"cyberfucked",
+	"cyberfucker",
+	"cyberfuckers",
+	"cyberfucking",
+	"damn",
 	"dick",
+	"dickhead",
 	"dildo",
+	"dildos",
+	"dink",
+	"dinks",
+	"dirsa",
+	"dlck",
+	"dogfucker",
+	"doggin",
+	"dogging",
+	"donkeyribber",
+	"doosh",
+	"duche",
 	"dyke",
+	"ejaculate",
+	"ejaculated",
+	"ejaculates",
+	"ejaculating",
+	"ejaculatings",
+	"ejaculation",
+	"ejakulate",
+	"fuck",
 	"fag",
+	"fagging",
+	"faggitt",
+	"faggot",
+	"faggs",
+	"fagot",
+	"fagots",
+	"fags",
+	"fanny",
+	"fannyflaps",
+	"fannyfucker",
+	"fanyy",
```

Comment:
```suggestion
```

A lot of these would be handled by other words already present, and `fanny`'s a bit of a peculiar one, because it's also a name.